guilt: set GIT_EXEC_PATH in guilt wrapper

git has the habit of tracking the directory where it was 
installed. If you build git, relocate git and then remove
the old directory --exec-path will return that original
(now non-existent) directory. We insist that git and 
guilt be in the same directory to ensure they are matched,
so we use the environment variable GIT_EXEC_PATH to 
force the issue.

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>

---
 guilt |    2 ++
 1 file changed, 2 insertions(+)

--- a/guilt
+++ b/guilt
@@ -23,6 +23,8 @@ esac
 # we change directories ourselves
 SUBDIRECTORY_OK=1
 
+export GIT_EXEC_PATH=`dirname $0`/../libexec/git-core
+
 if [ -z "$DO_NOT_USE_GITREPO_COMMANDS" ]; then
     . "$(git --exec-path)/git-sh-setup"
 fi
